dashboard/src/composables/format.ts

```diff
@@ -0,0 +1,34 @@
+import moment from "moment";
+
+export function FormatDateTime(value: Date | undefined) {
+  if (!value || Number.isNaN(value.getTime())) {
+    return "";
+  }
+  return moment(value).format("YYYY-MM-DD HH:mm:ss");
+}
```

Comment:
Here's a polished version:

```suggestion
export function formatDateTime(value?: Date): string {
  if (!(value instanceof Date) || isNaN(value.getTime())) {
    return "";
  }
  return moment(value).format("YYYY-MM-DD HH:mm:ss");
}
```